src/main/java/org/terasology/crafting/ui/CraftRecipeWidget.java

```diff
@@ -114,17 +115,21 @@ public void onDraw(Canvas canvas) {
         canvas.drawWidget(result, Rect2i.createFromMinAndSize(size.x - resultSize.x, 0, resultSize.x, resultSize.y));
     }
 
+
     @Override
-    public void update(float delta) {
+    public boolean onKeyEvent(NUIKeyEvent event) {
```

Comment:
This one made me stop for a moment - didn't compile. Figuring auto-refactor or copy paste from somewhere :-) Attempted a basic fix, but unsure how to test (where the quantity-modifier is active).

@MarcinSc where is this multiplier exercised? Can you check if it works after my tweak?

Same in other file, also fixed. Was able to craft just fine after the tweak at least.